tests/python/test_fields_builder.py

```diff
@@ -3,174 +3,226 @@
 
 import taichi as ti
 
+'''
+Test fields with shape.
+'''
+
 
 @ti.test(arch=[ti.cpu, ti.cuda, ti.vulkan, ti.metal])
 def test_fields_with_shape():
-    n = 5
-    x = ti.field(ti.f32, [n])
+    shape_size_1d = 5
+    x = ti.field(ti.f32, shape=shape_size_1d)
```

Comment:
Can we simply use `shape` instead of `shape_size_1d`?

Reply:
fixed